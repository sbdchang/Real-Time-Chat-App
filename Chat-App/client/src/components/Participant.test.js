import React from 'react';
import ReactDOM from 'react-dom';
import { render, screen } from '@testing-library/react';
import Participant from './Participant';
import renderer from 'react-test-renderer';
import MainView from './MainView';


//This function fails TODO
/*
test('renders participant', () => {
    const { getByLabelText } = render(<Participant />);
    const linkElement = getByText('participant');
    expect(linkElement).toBeInTheDocument();
  });
*/

it('Participant snapshot   ', () => {
  const tree = renderer.create(<MainView />).toJSON();
  expect(tree).toMatchSnapshot();
});


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Participant />, div);
});
