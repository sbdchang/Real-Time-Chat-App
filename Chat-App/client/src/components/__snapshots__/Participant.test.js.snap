// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant snapshot    1`] = `
<div
  className="participant"
>
  <video
    autoPlay={true}
  />
  <audio
    autoPlay={true}
    muted={true}
  />
</div>
`;
